CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Added
 
+- Support reading the Join Server's default JoinEUI and using this in the CLI for end device creation.
```

Comment:
- Please mention the actual API that was added
- Please mention the configuration that was added